ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/service/internal/RemoteFileServiceImpl.java

```diff
@@ -775,6 +941,7 @@ private long getContentLength(File reqFile, String destinationFormat, long start
         if (startCoordinate > 0 || endCoordinate > 0) {
             length = endCoordinate - startCoordinate;
         } else {
+            // TODO: figure out what happens here!
```

Comment:
This is for the AES files which are stored in the archive. `file length= (Header 16 bit) + (Actual file)` So we need to remove the header bit every time from the file size